object-manipulation-3/main.js

```diff
@@ -0,0 +1,68 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var deck = [];
+var suits = ['♠', '♥', '♣', '♦'];
+var ranks = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K', 'A'];
+var players = [
+  {
+    name: 'Sean Kim',
+    hand: []
+  },
+  {
+    name: 'Andrew Kim',
+    hand: []
+  },
+  {
+    name: 'Jay Park',
+    hand: []
+  },
+  {
+    name: 'Jacob Park',
+    hand: []
+  }
+];
+
+for (var i = 0; i < 4; i++) {
+  for (var j = 0; j < 13; j++) {
```

Comment:
Instead of hard-coding the boundaries, I would use the length property of the arrays you are looping over.